docs_theme/main.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% block scripts %}
+{{ super() }}
+<script>
+  fetch('{{ 'versions.txt'|url }}')
+    .then(function(res) {if (res.ok) return res.text();})
+    .then(function(text) {return text.trim().split("\n");})
+    .then(function(versions) {
+      const version_div = document.createElement("div");
+      version_div.classList.add("version");
+      version_div.appendChild(document.createTextNode("Version: "));
+      const version_select = document.createElement("select");
+      version_select.classList.add("x9c4-custom-select");
+      version_select.id = "version-select";
+      version_div.appendChild(version_select);
+      const option = document.createElement("option");
+      option.innerText = "latest";
+      option.value = "";
+      version_select.add(option);
+      versions.map(function(version) {
+        const option = document.createElement("option");
+        option.innerText = version;
+        option.value = '/en/' + version;
+        version_select.add(option);
+      });
+      {% if config.theme.version.endswith("dev") %}
+      version_select.value = "";
+      {% else %}
+      version_select.value = "{{ config.theme.version }}";
+      {% endif %}
+      version_select.onchange = function(event) { window.location = '{{ '.'|url }}' + event.target.value };
+      console.log(version_select)
```

Comment:
Do you want to leave this log statement in?

Reply:
Nope...